mavros/src/plugins/actuator_control.cpp

```diff
@@ -17,6 +17,7 @@
 #include <mavros/mavros_plugin.h>
 
 #include <mavros_msgs/ActuatorControl.h>
+#include <mavros_msgs/ActuatorControlTarget.h>
```

Comment:
According to the MAVLINK Common message set, https://pixhawk.ethz.ch/mavlink/#ACTUATOR_CONTROL_TARGET, only `ACTUATOR_CONTROL_TARGET` exists at the moment. So I will replace the `ActuatorControl` with `ActuatorControlTarget` only.